src/Symfony/Bundle/FrameworkBundle/Command/WorkflowDumpCommand.php

```diff
@@ -37,6 +37,7 @@ protected function configure()
             ->setDefinition(array(
                 new InputArgument('name', InputArgument::REQUIRED, 'A workflow name'),
                 new InputArgument('marking', InputArgument::IS_ARRAY, 'A marking (a list of places)'),
+                new InputOption('workflow-name-label', null, null, 'Labels graph with workflow name'),
```

Comment:
what about `--label | --label="value"` api? Using workflow name as default value.

Reply:
Thx for your comment thats a very good idea. I will change it.

Reply:
I updated my pull request now it's like you suggested